### PR TITLE
Setting up branch for release-1.6.4

### PR DESCRIPTION
**Pull request by akshayhavile**

### CODE REVIEW

File: docs/install/helm.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ kubectl apply -f <output_dir>/ako/crds/
 helm list -n avi-system
 
 NAME          	NAMESPACE 	REVISION	UPDATED                             	    STATUS  	CHART    	APP VERSION
-ako-1593523840	avi-system	1       	2020-09-16 13:44:31.609195757 +0000 UTC	    deployed	ako-1.6.3	1.6.3
+ako-1593523840	avi-system	1       	2020-09-16 13:44:31.609195757 +0000 UTC	    deployed	ako-1.6.4	1.6.4
```
Comment (swathinsankaran) [inline docs/install/helm.md:97]: The list command will give the older version say 1.6.3 I guess.

Comment (akshayhavile) [reply docs/install/helm.md]: Addressed

**Review by aaha97 — APPROVED** [review]

**Comment by anish-avi** [comment]
AKO PR Sanity Check succeeded

**Comment by akshayhavile** [comment]
build ako

**Comment by anish-avi** [comment]
AKO PR Sanity Check succeeded

**Comment by anish-avi** [comment]
AKO PR Sanity Check failed

**Comment by anish-avi** [comment]
AKO PR Sanity Check failed

**Comment by akshayhavile** [comment]
testing-akshay

**Review by swathinsankaran — APPROVED** [review]

**Pull request merged**